.github/workflows/tests.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     runs-on: ubuntu-22.04
     strategy:
       matrix:
-          python-version: ["3.8", "3.9", "3.10", "3.11"]
+          python-version: ["3.8", "3.9", "3.10"]
```

Comment:
I didn't see questions about that change in galter-subjects-utils, but I will explain anyway :smile: : there is a dependency (cchardet) in version 11 of invenio-app-rdm that doesn't build with python 11 because the dependency is unmaintained anymore. Latest version of invenio-previewer switches to a maintained/API compatible dependency, but that version of invenio-previewer is not pulled by invenio-app-rdm v11, so we remove our support for 3.11 as well.